specs/eip4844/p2p-interface.md

```diff
@@ -113,8 +113,6 @@ Alias `sidecar = signed_blobs_sidecar.message`.
 - _[IGNORE]_ The sidecar is the first sidecar with valid signature received for the `(proposer_index, sidecar.beacon_block_slot)` combination,
   where `proposer_index` is the validator index of the beacon block proposer of `sidecar.beacon_block_slot`
 
-Note that a sidecar may be propagated before or after the corresponding beacon block.
-
 Once both sidecar and beacon block are received, `validate_blobs_sidecar` can unlock the data-availability fork-choice dependency.
```

Comment:
```suggestion
Once the sidecar and beacon block are received together, `validate_blobs_sidecar` can unlock the data-availability fork-choice dependency.
```
Nit: if we couple them here then we should describe it as receiving the two together instead of independently.